onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -359,14 +368,19 @@ def run_pytorch(
                 if max_input_size is not None and sequence_length > max_input_size:
                     continue
 
-                logger.info("Run PyTorch on {} with input shape {}".format(model_name, [batch_size, sequence_length]))
-                input_ids = torch.randint(
-                    low=0,
-                    high=config.vocab_size - 1,
-                    size=(batch_size, sequence_length),
-                    dtype=torch.long,
-                    device=device,
-                )
+                if config.model_type == "vit":
+                    logger.info("Run PyTorch on {} with input shape {}".format(model_name, [batch_size, 3, 224, 224]))
+                    input_ids = torch.randn(size=(batch_size, 3, 224, 224),dtype=torch.float, device=device)
```

Comment:
This use float32 inputs. Could we support float16 input when precision == Precision.FLOAT16?

Reply:
Good catch, that might explain the slow float16 performance I saw when testing with Pytorch. Will test more with it.